src/components/scheduling/goals/SchedulingGoal.svelte

```diff
@@ -13,7 +13,6 @@
   import { tooltip } from '../../../utilities/tooltip';
   import Collapse from '../../Collapse.svelte';
   import ContextMenuItem from '../../context-menu/ContextMenuItem.svelte';
-  import Input from '../../form/Input.svelte';
```

Comment:
Not a criticism, but just curious. Is there a reason why we're moving away from the `Input` component?

Reply:
It's been a second since I made this change but I think it was actually a style fix, maybe related to padding